modules/secrets-store-aws.adoc

```diff
@@ -0,0 +1,370 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+//
+// IMPORTANT: This file requires you to define :secrets-store-provider: before including this module.
+
+ifeval::["{secrets-store-provider}" == "AWS Secrets Manager"]
+:aws-secrets-manager:
+endif::[]
+ifeval::["{secrets-store-provider}" == "AWS Systems Manager Parameter Store"]
+:aws-systems-manager-parameter-store:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="secrets-store-aws_{context}"]
+= Mounting secrets from {secrets-store-provider}
+
+You can use the {secrets-store-operator} to mount secrets from {secrets-store-provider} to a CSI volume in {product-title}. To mount secrets from {secrets-store-provider}, your cluster must be installed on AWS and use AWS Security Token Service (STS).
+
+[IMPORTANT]
+====
+It is not supported to use the {secrets-store-operator} with {secrets-store-provider} in a hosted control plane cluster.
+====
+
+.Prerequisites
+
+* Your cluster is installed on AWS and uses AWS Security Token Service (STS).
+* You have installed the {secrets-store-operator}. See _Installing the {secrets-store-driver}_ for instructions.
+* You have configured {secrets-store-provider} to store the required secrets.
+* You have extracted and prepared the `ccoctl` binary.
+* You have installed the `jq` CLI tool.
```

Comment:
Is `jq` required here? Or can it be replaced with `jsonpath` (which I think is preferred to jq and can accomplish pretty much the same things)?

Reply:
Same answer as the other comment on jq